### PR TITLE
chore: treat begin stream event

### PR DESCRIPTION
**Pull request by r4mmer**
### Acceptance Criteria

- Treat begin stream event with no action.

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: src/sync/stream.ts [hunk 1]
```diff
@@ -402,8 +412,10 @@ function buildListener(manager: StreamManager, resolve: () => void) {
       );
       return;
     }
-    // Vertex is a transaction in the history of the last address received
-    if (isStreamSyncHistoryVertex(wsData)) {
+    if (isStreamSyncHistoryBegin(wsData)) {
+      manager.logger.info('Begin stream event received.');
```
Comment (msbrogli) [inline src/sync/stream.ts:416]: Does it log the stream id?

File: __tests__/stream.test.ts [hunk 2]
```diff
@@ -226,4 +249,32 @@ describe('Websocket stream history sync', () => {
       mockServer.stop();
     }
   }, 10000);
+
+  it('should ignore unknown stream ids', async () => {
```
Comment (tuliomir) [inline __tests__/stream.test.ts:253]: question: I'm under the impression that even if the wallet fails to handle the unknown stream id we would not be able to validate this in this test. Is there a more explicit way to check the results of these unknown stream ids?

File: src/sync/stream.ts (hunk 1 above)
Comment (r4mmer) [reply src/sync/stream.ts:416]: No, but we can include it if you want.

File: __tests__/stream.test.ts (hunk 2 above)
Comment (tuliomir) [reply __tests__/stream.test.ts:253]: As discussed, this test provides enough confidence on the results of the websocket with the balance validation.
Future possibilities can be discussed later.

**Review by tuliomir — APPROVED** [review]

**Pull request merged**